setup.py

```diff
@@ -58,6 +58,8 @@
     'cc-plugin-imos>=1.3.0',
     'fiona>=1.8.8',
     'matplotlib>=3.0.3',
+    'pandas>=0.24.2,<0.25.0',
```

Comment:
These lines are unnecessary. We just need to add constraints to the *environment* (as you've done in requirements.txt/constraints.txt), not add dependencies to the *package*. The reason is that it already gets these dependencies via the aodntools package. Having these here as well just means it will require synchronising updates to these versions with the aodntools package, which is a pain when it comes to updating/deploying them if they get out of sync.